docs/First-Installation.md

```diff
@@ -21,65 +21,57 @@ The simplest way to get most of these components is through an all-in-one kit su
 
 ### Required
 
- * [Raspberry Pi](https://retropie.org.uk/about/building/)
- * MicroSD Card ([list of compatible SD cards](http://elinux.org/RPi_SD_cards))
- * Screen (TV, computer monitor, projector, etc) - anything with HDMI or RCA
- * Video cable
-   * Pi 4 will need a [Micro HDMI to HDMI](https://user-images.githubusercontent.com/540857/107463741-c3a47480-6b2c-11eb-8402-7313a490e234.jpg) cable
-   * Pi 1, 2, and 3 will need a full-size [HDMI](https://user-images.githubusercontent.com/540857/107463845-ff3f3e80-6b2c-11eb-8031-3d0ed5563861.jpg) cable
-   * Pi Zero will need a [Mini HDMI to HDMI](https://user-images.githubusercontent.com/540857/107686191-e3de4b80-6c72-11eb-8162-07a82afaac7b.jpg) cable
-   * [4-Pole RCA to 3.5mm](https://user-images.githubusercontent.com/540857/107688722-f3ab5f00-6c75-11eb-9837-ffe7f385b1c5.jpg) is also an option for older screens
- * Power supply
-   * View the official [Raspberry Pi Power](https://www.raspberrypi.org/documentation/hardware/raspberrypi/power/) documentation for each model
- * Game controller of your choice 
-   * Can be USB-wired, wireless (with a dongle), or Bluetooth (with or without a dongle. Pi 3 and later models have built-in bluetooth and won't need a dongle)
-   * The [Control Block](http://blog.petrockblock.com/2014/12/29/controlblock-power-switch-and-io-for-the-raspberry-pi/) can use original SNES controllers
+* [Raspberry Pi](https://retropie.org.uk/about/building/)
+* MicroSD Card ([list of compatible SD cards](http://elinux.org/RPi_SD_cards))
+* [USB MicroSD card reader](https://user-images.githubusercontent.com/540857/107463579-68728200-6b2c-11eb-9612-05c4d1931882.jpg) - an adapter that allows plugging the MiscroSD card into any USB port
```

Comment:
typo "MiscroSD" but IMO title should be "MicroSD card reader" and instead
> * MicroSD card reader ([Example #1](https://user-images.githubusercontent.com/540857/107868575-6f163780-6e53-11eb-9943-faec57cbf48e.png), [example #2](https://user-images.githubusercontent.com/540857/107868592-94a34100-6e53-11eb-91ed-e645be9827aa.jpg)) - If your computer/laptop does not have an MicroSD card slot, acquire an [external USB MicroSD card reader](https://user-images.githubusercontent.com/540857/107463579-68728200-6b2c-11eb-9612-05c4d1931882.jpg).

It just reads better to me to have the no-effort part before the effort part :)